go.mod

```diff
@@ -4,6 +4,7 @@ go 1.22.3
 
 require (
 	github.com/go-logr/logr v1.4.1
+	github.com/gorilla/mux v1.8.1
```

Comment:
You can use routing from std lib http package instead:
https://go.dev/doc/go1.22#enhanced_routing_patterns